enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md

```diff
@@ -151,6 +151,11 @@ type PolicyRootOfTrust struct {
 	// https://github.com/sigstore/fulcio and https://github.com/sigstore/rekor
 	// +optional
 	FulcioCAWithRekor *FulcioCAWithRekor `json:"fulcioCAWithRekor,omitempty"`
+
+  // PKI defines the root of trust based on Root CA(s) and corresponding intermediate certificates.
+  // +optional
+  PKI *PKI `json:"pki,omitempty"`
```

Comment:
nit: seems like inconsistent indent whitespace vs. the previous `FulcioCAWithRekor` entry?

Reply:
Updated thanks.

Reply:
(Still outstanding I think.)